### PR TITLE
repo install instructions

### PR DESCRIPTION
**Pull request by nottrobin**
QA
--

Try out the new [readme instructions](https://github.com/nottrobin/docs.ubuntu.com/tree/43-repo-install-instructions).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,50 @@
...
 A central platform for hosting Ubuntu's various documentation portals.
 
-Basic usage
----
+## Local development
+
+### Dependencies
+
+1. [Install Docker](https://www.docker.com/community-edition#/download)
+2. Get [the `repo` command](https://github.com/esrlabs/git-repo), e.g.:
```
Comment (caldav) [inline README.md]: This whole section can be replaced by `snap install git-repo` (upon install, `git-repo.repo` is automatically aliased to the `repo` command).

Comment (nottrobin) [inline README.md]: Ah it's snapped! Awesome.

Comment (nottrobin) [inline README.md]: Done?

Comment (caldav) [inline README.md]: Never tried the Docker snap for real things, needs testing I guess, but repo works fine

Comment (nottrobin) [inline README.md]: @WillMoggridge uses the docker snap, so I think it works okay.

Comment (WillMoggridge) [inline README.md]: It works mostly fine... Just trouble configuring DNS at work and existing solutions don't seem to work.
Works with no problems at home.

Comment (WillMoggridge) [inline README.md]: git-repo causes permission problems for me. :(
@caldav @nottrobin 

```
warn: cannot enable ssh control master for git@github.com:None  
[Errno 13] Permission denied                                    
fatal: cannot exec 'ssh': Permission denied                     
fatal: unable to fork                                           
```

Comment (nottrobin) [inline README.md]: what a pain. I guess I should try it out myself, probably :)

Comment (nottrobin) [inline README.md]: @WillMoggridge permission issues should be sorted - repo docker container doesn't have your SSH creds, so I switch it to always use HTTPS for Git pulling

**Comment by nottrobin** [comment]
Fixes #43

File: README.md (hunk 1 above)
Comment (WillMoggridge) [inline README.md]: Ah yes, that explains it

**Pull request merged**